database/seeds/AddCollectionTypeIdSeeder.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+use Illuminate\Database\Seeder;
+use \Jitterbug\Models\Collection;
+
+class AddCollectionTypeIdSeeder extends Seeder
+{
+    /**
+     * Run the database seeds.
+     *
+     * @return void
+     */
+    public function run()
+    {
+      $uarmsId = 3;
```

Comment:
Is it correct that this migration would only work for UNC, while the CollectionTypesSeeder could seed sample data for any user?

Reply:
The data in this seeder only applies to UNC, but the structure of the seeder could be used by other institutions if they substitute their IDs and collection ID ranges.